plugin/evm/atomic/atomic_trie_iterator_test.go

```diff
@@ -32,7 +32,7 @@ func TestIteratorCanIterate(t *testing.T) {
 
 	// create an atomic trie
 	// on create it will initialize all the transactions from the above atomic repository
-	atomicBackend, err := NewAtomicBackend(db, utils.TestSharedMemory(), nil, repo, lastAcceptedHeight, common.Hash{}, 100)
+	atomicBackend, err := NewAtomicBackend(db, utils.TestSnowContext().SharedMemory, nil, repo, lastAcceptedHeight, common.Hash{}, 100)
```

Comment:
nit: can factor out `snowCtx := utils.TestSnowContext()` (or `ctx`, but I think we should start using `ctx` only for `context.Context`